templates/microApp/package.json

```diff
@@ -1,5 +1,35 @@
 {
-    "scripts": {
-      "dummy": "dummy command"
-    }
-  }
\ No newline at end of file
+  "name": "micro-app-demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --mode production",
+    "build:dev": "webpack --mode development",
+    "build:start": "cd dist && PORT=8080 npx serve",
+    "start": "webpack serve --open --mode development",
+    "start:live": "webpack serve --open --mode development --liveReload"
+  },
+  "devDependencies": {
+    "@babel/core": "7.13.15",
```

Comment:
@achaljain for dependencies like linting etc we should be using TurboRepo config package